benchmarks/rosa.fpcore

```diff
@@ -212,10 +212,9 @@
 (FPCore (a b c)
   :name "triangle"
   :cite (darulova-kuncak-2014)
-  :pre (and (<= 4.71 b 4.89) (<= 4.71 c 4.89))
-  (let ([a 9])
-    (let ([s (/ (+ (+ a b) c) 2)])
-      (sqrt (* (* (* s (- s a)) (- s b)) (- s c))))))
+  :pre (and (<= 9.0 a 9.0) (<= 4.71 b 4.89) (<= 4.71 c 4.89))
```

Comment:
Any reason to leave this as a free variable restricted to [9, 9] instead of making it a let-bound variable?

Reply:
This is just for consistency with the other cases, if we let-define `a` then it can't be an argument as well.